demos/react-jsnation/src/components/DataTable.tsx

```diff
@@ -1,7 +1,7 @@
 import { Cell, FormulaFn } from "@starbeam/core";
 import { LOGGER, LogLevel } from "@starbeam/debug";
 import { useProp, useReactiveSetup } from "@starbeam/react";
-import { DevTools } from "@starbeamx/devtool";
+// import { DevTools } from "@starbeamx/devtool";
```

Comment:
Commented out to temporarily remove the devtool integration due to the JSX Runtime issue.

Reply:
Does this still need to be commented out now that we're running the prod tests via each workspace separately?

Reply:
I believe this is still needed to have the vitest runs pass at all, even for the non-prod test runs.